charreplace.go

```diff
@@ -10,22 +10,48 @@
 package main
 
 import (
-  "bufio"
   "bytes"
+  "bufio"
   "fmt"
   "io/ioutil"
   "os"
   "path/filepath"
+	"strings"
   "time"
 )
 
 var count = 0
+var runpath = "./"
 
 func main() {
   start := time.Now()
 
+
+	if(len(os.Args) <= 1) {
+		var confirm string
+		dir, _ := os.Getwd()
+		fmt.Printf("Warning: This action will possibly corrupt files in the directory %s", dir)
+		fmt.Printf("Continue [y,n] ")
+		fmt.Scanln(&confirm)
+		if(strings.ToLower(confirm) != "y") {
+			fmt.Printf("Aborting!")
```

Comment:
It would be nice if this were a `Println` instead of `Printf` as it looks slightly better in the console.